### PR TITLE
Fix failing tests

### PR DESCRIPTION
**Pull request by ngrinkevich**
A couple of adjustments to help with `increaseTimeTo()` inconsistency.

### CODE REVIEW

**Comment by ngrinkevich** [comment]
@pdyraga ready!

File: solidity/test/token_stake/TestTokenStake.js [hunk 1]
```diff
@@ -801,7 +801,7 @@ contract('TokenStaking', function(accounts) {
         {from: ownerOne}
       );
 
-      await increaseTimeTo(createdAt + initializationPeriod + delegationTime + 1)
+      await increaseTimeTo(createdAt + initializationPeriod + delegationTime + timeRoundMargin)
```
Comment (pdyraga) [inline solidity/test/token_stake/TestTokenStake.js]: Hmm... this is weird, shouldn't margin apply to `-` cases only?

Comment (ngrinkevich) [reply solidity/test/token_stake/TestTokenStake.js]: yeah it's weird especially if you can't replicate it locally 😅 My only guess here I think if it was a normal transaction next (not .call like we have) then +1 would work reliably

**Comment by ngrinkevich** [comment]
closing this, https://github.com/keep-network/keep-core/pull/1563 has a proper fix

**Pull request closed without merging**